.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Spellcheck with cspell'
+on:
+  pull_request:
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: streetsidesoftware/cspell-action@v5
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Doesn't this run with the default value?

Reply:
It may, I'll check. This was my crash course in how tokens and actions work together